.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
```

Comment:
I'm curious if we can just use the default image? Have you tried it? If not, you should. If you have, you should add a comment saying why it's necessary.